wsmaster/che-core-api-workspace/src/test/java/org/eclipse/che/api/workspace/server/WorkspaceConfigJsonAdapterTest.java

```diff
@@ -1,163 +0,0 @@
-/*
- * Copyright (c) 2012-2017 Red Hat, Inc.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *   Red Hat, Inc. - initial API and implementation
- */
-package org.eclipse.che.api.workspace.server;
-
-import static org.eclipse.che.api.core.model.workspace.config.MachineConfig.MEMORY_LIMIT_ATTRIBUTE;
-import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertNotNull;
-import static org.testng.Assert.assertTrue;
-
-import com.google.common.io.CharStreams;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
-import java.io.File;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.Reader;
-import java.net.URL;
-import org.eclipse.che.api.workspace.server.adapter.WorkspaceConfigJsonAdapter;
-import org.mockito.testng.MockitoTestNGListener;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Listeners;
-import org.testng.annotations.Test;
-
-/**
- * Tests of {@link WorkspaceConfigJsonAdapter}.
- *
- * @author Yevhenii Voevodin
- */
-@Listeners(MockitoTestNGListener.class)
-public class WorkspaceConfigJsonAdapterTest {
-
-  private static final String INVALID_CONFIGS_DIR_NAME = "invalid_configs";
```

Comment:
Should you remove [these resources](https://github.com/eclipse/che/tree/removeche4support/wsmaster/che-core-api-workspace/src/test/resources/invalid_configs) too or they are still used by another test?

Reply:
done